portfolio/src/main/webapp/index.html

```diff
@@ -6,12 +6,16 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getComments()">
+  <body>
     <div id="name">
       <h1>George Zheng</h1>
     </div>
     <hr>
+    <button type = "button" onclick = "deleteComments()">Delete Comments</button>
     <ul id="comments-history"></ul>
+    <p>Enter number of comments to view:</p>
+    <input type="number" onchange="getComments(this.value)">
+    <button type = "button">View Comments</button>
```

Comment:
I think putting the input and button inside a `<form>` looks better?

Reply:
Wrapped these up into a form